agents/errors.go

```diff
@@ -20,6 +20,9 @@ var (
 	// ErrInvalidChainReturnType is returned if the internal chain of the agent returns a value in the
 	// "text" filed that is not a string.
 	ErrInvalidChainReturnType = errors.New("agent chain did not return a string")
+	// ErrInvalidLLMGenerationsMessageReturn is returned if the internal llm of the agent returns a value
```

Comment:
Sorry for taking so long to reply.
After checking the code, the error does not occur here because if the message is null, the error is handled in advance and does not go to this logic

Reply:
so I deleted it